decidim-core/app/assets/stylesheets/decidim/modules/_collapsible-list.scss

```diff
@@ -36,7 +36,7 @@
 
   .collapsible-list__see-more{
     display: none;
-    color: #599aa6;
+    color: $secondary;
```

Comment:
```suggestion
    color: var(--secondary);
```

Reply:
@microstudi is this suggestion correct? did you meant it?
cc/ @ivan-mr @SilviaGrrr

Reply:
no, at least not for the moment, just bringing out that this kind of colors should be customizable as well from the admin side using the va(--). However it should be changed in many other places and it is out of scope of this PR.

Reply:
See the next comment @tramuntanal